.JamesLau-fe-project/src/Components/ArticlePage.jsx

```diff
@@ -1,33 +1,47 @@
 import { useState, useEffect } from "react";
 import { useParams } from "react-router-dom";
-import { getArticleById } from "../api";
-import Comments from "./Comments"
+import { getArticleById, patchVotesByArticleId } from "../api";
+
+import Comments from "./Comments";
 const ArticlePage = () => {
   const { article_id } = useParams();
   const [article, setArticle] = useState({});
   const [loading, setLoading] = useState(true);
+  const [currentVotes, setCurrentVotes] = useState(0);
 
   useEffect(() => {
     getArticleById(article_id).then((data) => {
       setArticle(data);
+      setCurrentVotes(data.votes);
       setLoading(false);
     });
   }, [article_id]);
```

Comment:
This doesn't need to be here, the article id won't change unless you navigate to another page. The array must exist, but doesn't always need elements.